xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/service/BittrexTradeService.java

```diff
@@ -106,10 +123,25 @@ public OpenOrdersParams createOpenOrdersParams() {
     try {
       List<Order> orders = new ArrayList<>();
       for (String orderId : orderIds) {
-        BittrexOrder order = getBittrexOrder(orderId);
-        if (order != null) {
-          LimitOrder limitOrder = BittrexAdapters.adaptOrder(order);
-          orders.add(limitOrder);
+        BittrexOrder bittrexOrder = getBittrexOrder(orderId);
+        if (bittrexOrder != null) {
```

Comment:
rather then nest this so much can you just return on the negation. e.g. 
```
if (bittrexOrder != null) {
    continue;
}
if (!(order instanceof MarketOrder)) {
    continue;
}
```

This makes following the code a bit easier to read.

Also instead of executions.size() use !isEmpty();

Reply:
do you see what I'm asking there I pasted code here? Instead of nesting the if statements, just continue so you don't need to indent so much, makes the code more readable.

Reply:
I have added the continue when checking if the bittrexOrder is null, I agree it is more readable, for checking if the order is a MarketOrder, I think it is good like that, because fetching the price only has to be done for MarketOrders, otherwise just add to the list